helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -710,7 +726,9 @@ protected Message createMessage(Message.MessageType type, String msgId, String f
   }
 
   @AfterClass
-  public void cleanupLiveInstanceOwners() {
+  public void cleanupLiveInstanceOwners() throws InterruptedException {
```

Comment:
Will this method being called by all sub test classes?

Reply:
All the ones using ZkTestBase